src/index.js

```diff
@@ -52,7 +57,7 @@ function parseValue(value) {
   } else if (_.isPlainObject(value)) {
     return parseMap(value);
   } else {
-    return value;
+    return parseEndValue(value);
```

Comment:
There is already a `parseValue` function, thats why I chose `parseEndValue`... :sleeping: :zzz: